Makefile

```diff
@@ -415,3 +415,16 @@ test-app-dev: tmp/.ekam-run
 	@cp src/sandstorm/test-app/test-app.capnp tmp/sandstorm/test-app/test-app.capnp
 	@cp src/sandstorm/test-app/*.html tmp/sandstorm/test-app
 	spk dev -Isrc -Itmp -ptmp/sandstorm/test-app/test-app.capnp:pkgdef
+
+# ====================================================================
+# meteor-testapp.spk
+
+meteor-spk-0.3.2/meteor-spk:
+	@$(call color,downloading meteor-spk)
+	@curl https://dl.sandstorm.io/meteor-spk-0.3.2.tar.xz | tar Jxf -
+
+tests/assets/meteor-testapp.spk: meteor-testapp meteor-spk-0.3.2/meteor-spk $(shell find meteor-testapp -type f -o -type d)
+	@cd meteor-testapp && PATH="$$PATH:$$PWD/bin" && ../meteor-spk-0.3.2/meteor-spk pack ../tests/assets/meteor-testapp.spk
```

Comment:
You need to put the `-I../src` after `pack`. `spk` allows you to put it on either side, but the `meteor-spk` wrapper will get confused if you put it before.

Specifically what's happening here is the `meteor-spk` script looks at the first argument to decide what extra setup it needs to do before running `spk`. In the case of `pack` it arranges the dependencies where `spk` can find them. But in your case it sees `-I../src` which it doesn't understand, so it just passes along to `spk` verbatim, but then `spk` can't find the files it needs since `meteor-spk` didn't set them up.